package.json

```diff
@@ -17,10 +17,11 @@
     "build:docs:settlement": "jsdoc2md lib/settlement/settlement.js > Docs/settlement.md",
     "build:docs:driip-settlement": "jsdoc2md lib/settlement/driip-settlement.js > Docs/driip-settlement.md",
     "build:docs:null-settlement": "jsdoc2md lib/settlement/null-settlement.js > Docs/null-settlement.md",
-    "test": "nyc mocha 'lib/**/*.spec.js' --exit",
+    "test": "NODE_PATH=$npm_package_peerInstallOptions_prefix/node_modules nyc mocha 'lib/**/*.spec.js' --exit",
     "test:watch": "npm run test -- --watch",
     "lint": "eslint --ignore-path .gitignore .",
-    "lint:fix": "npm run lint -- --fix"
+    "lint:fix": "npm run lint -- --fix",
+    "postinstall": "bash -c \"if [[ '$npm_config_production' == 'true' || '$npm_config_only' == prod* ]] ; then exit 0 ; else npx npm-install-peers ; fi\""
```

Comment:
Good that it was possible to incorporate the peer install requirement in the post install script! :)

Reply:
The complicating factor is to make it work consistently on both macos, linux and on the TC agent images... trial and error :(